terraform/error.go

```diff
@@ -0,0 +1,9 @@
+package terraform
+
+import "fmt"
+
+// Errors that might be returned from procesing the HCL
+var (
+	ErrNoQueries       = fmt.Errorf("no terraform entities found, looks empty")
```

Comment:
Means empty or that we do not support any of the resources on the Plan/HCL?

Reply:
Means no queries were found - eg something generates a diff where there is nothing to process or contains no files.